grpc-api/src/main/java/io/deephaven/grpc_api/session/TicketRouter.java

```diff
@@ -70,6 +85,7 @@ public TicketRouter(final Set<TicketResolver> resolvers) {
      * @param <T> the expected return type of the ticket; this is not validated
      * @return an export object; see {@link SessionState} for lifecycle propagation details
      */
+    //TODO #412 use this or remove it?
```

Comment:
we can drop this TODO, too (and the ones below; or again, I will look for and clear these)